src/System.Management.Automation/engine/MergedCommandParameterMetadata.cs

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft Corporation.
```

Comment:
@bergmeister, your last commit had 1 failures in `PowerShell-CI-windows`
WSMan Config Provider tab complete tests.Tab completion gets dynamic parameters for 'localhost\plugin' using '-fi'
```powershell
Expected strings to be the same, but they were different.
String lengths are both 12.
Strings differ at index 4.
Expected: 'FileNameFile'
But was:  'FileFileName'
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\TabCompletion\TabCompletion.Tests.ps1: line 1406
1406:         $completionOptions | Should -BeExactly ([string]::Join("", $expected))
```